plugins/rapid7_intsights/plugin.spec.yaml

```diff
@@ -2,20 +2,19 @@ plugin_spec_version: v2
 extension: plugin
 products: [insightconnect]
 name: rapid7_intsights
-title: Rapid7 IntSights
-description: Rapid7 IntSights is disrupting external threat intelligence with a combination of human and automated collection, intelligent analysis, and strategic threat hunting that turns the clear, deep, and dark webs into an intelligence resource that any company can deploy
-version: 3.2.0
+title: Threat Command
+description: Threat Command by Rapid7 is disrupting external threat intelligence with a combination of human and automated collection, intelligent analysis, and strategic threat hunting that turns the clear, deep, and dark webs into an intelligence resource that any company can deploy
```

Comment:
Quick question, is it supposed to be Threat Command by Rapid7 or just Threat Command?

Reply:
Threat Command by Rapid7 is the approved wording for this description.